setup.py

```diff
@@ -18,11 +19,26 @@ def finalize_options(self):
         _bdist_wheel.finalize_options(self)
 
 
+resources_globs = [
+    "*.h",
+    "*.idl",
+    "commit-sha",
+]
+if sys.platform.startswith("win"):
+    resources_globs.append("*.dll")
+elif sys.platform.startswith("linux"):
+    resources_globs.append("*.so")
+elif sys.platform.startswith("darwin"):
+    resources_globs.append("*.dylib")
+else:
+    raise RuntimeError(f"Platform '{sys.platform}' not supported")
```

Comment:
I'm not sure if we want to raise here too. We should raise in the download-script, because there simply aint any binaries for platforms other than win, linux and darwin. But ppl may want to install `wgpu-py` on e.g. a raspberry Pi and somehow provide their own `wgpu-native` libs.

Reply:
Awesome, I hadn't considered that

Reply:
It's besides the point, but on raspberry sys.platform apparently returns "linux" :)

I removed the raise from setup.py and rs.py

Reply:
mmm ... so we might need a more sophisticated solution eventually. Though for now this should be enough.

Reply:
there's just tons of constraints that the current python package management solutions (pip, pypi, etc) can't handle. e.g. c-compiler used (musl, gcc) cpu architecture (ARM...) and many more are all factors that we can't specifically build wheels for today. I wouldn't try to solve that here locally, it's a much bigger problem.